### DISCUSSION TITLE
Single View Runtime Section

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#242] [Ideas]
### Which area of this project could be most improved?

Fast Data

### Description

I would like to have a section in the runtime area of the console where:

* is possible to check, for each single view, the `number of projection changes` that still need to be processed by the `SVC` (i.e. the number of single views with a projection change record having the array `changes` with at least one record having `{state: "NEW"}`)

* is possible to trigger manually an operation of `full refresh`, i.e. a generation of all the single views starting from the projection changes computed from the base projection

* _Nice to have_: the lag for each projection that generates a particular single view 

This area would be useful to let the user manage the fast data flow  within the console, without using different tools (such as Grafana or Mongodb)

In particular, the operation of full refresh and PC monitoring at this moment is done manually with Mongo Db using technical users gathered from the env section of the console, which is not a best practice in terms of security

**Comment by TheRusCompass** [comment]
Hi @giovf96! Thanks a lot for your proposal! This is a very interesting one, we will include it in our roadmap

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
